test/unit/mock.py

```
@@ -425,35 +425,33 @@ def patch_console(self, console=None):
         old_console = sap.cli.core.set_console(console)
```

Flake8 (7.3.0)

[error] local variable 'old_console' is assigned to but never used

(F841)